content/tensorman.md

```diff
@@ -41,7 +41,13 @@ The user account working with Tensorman must be added to the `docker` group if t
 sudo usermod -aG docker $USER
 ```
 
-If Docker was just installed, then a reboot will be needed before Tensorman can be used.
+THe last thing is to add a kernel parameter:
```

Comment:
`THe` looks like an extra character is capitalized. I might also suggest "last step" instead of "last thing."

Might want to place an ellipses at the beginning of the sentence after the code block, since it's picking up the sentence from here.

Reply:
Does it work now?

Reply:
I don't know why you went with "one last thing" instead of the suggested change. The problem was the ambiguity of the word "thing."

Reply:
I see that "Then you are ready for liftoff!" is being added by this PR too. Are you intentionally trying to introduce an unprofessional/casual voice?

Reply:
> I don't know why you went with "one last thing" instead of the suggested change. The problem was the ambiguity of the word "thing."
Oh i just misread your message thats all. Either way its semantics at this point.

> I see that "Then you are ready for liftoff!" is being added by this PR too. Are you intentionally trying to introduce an unprofessional/casual voice?

:shrug: All of our messaging elsewhere is pretty casual, Is there a reason support-articles needs to be super professional?

Reply:
> Is there a reason support-articles needs to be super professional?

So people can take them seriously?

I moved the first one back closer to what you started with but replaced "thing" with "step," and contracted "you are" into "you're" in the final sentence since you're going for casual.

Reply:
:+1: